client/init.c

```diff
@@ -220,12 +220,13 @@ void printUsage(char* appName)
 		"  -l             - output non zero balances of all accounts\n"
 		"  -m N           - use N CPU mining threads (default is 0)\n"
 		"  -p ip:port     - public address of this node\n"
-		"  -P ip:port:CFG - run the pool, bind to ip:port, CFG is miners:fee:reward:direct:maxip:fund\n"
+		"  -P ip:port:CFG - run the pool, bind to ip:port, CFG is miners:maxip:maxconn:fee:reward:direct:fund\n"
```

Comment:
It seams pool owners have to change the scripts to adapt such config. Need address it in change log.